### PR TITLE
Add section on bundling dependent claims.

### PR DESCRIPTION
**Pull request by msporny**
Submitting patch request from David Chadwick on bundling dependent claims. We still need to discuss mitigation strategies.

/cc @David-Chadwick @dlongley @talltree @nage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2035,10 +2035,30 @@
 
       <section>
         <h3>Bundling Dependent Claims</h3>
+        <p>
+It is considered a best practice for issuers to atomize information in a
+credential, or use a signature scheme that allows for selective disclosure.
+In the former case, if the atomization is not done securely by the issuer,
+the holder may bundle together different credentials in a way that was not
```
Comment (burnburn) [inline index.html]: Suggest 'may' -> 'might' or 'might attempt to' to avoid confusion with normative MAY.

Comment (David-Chadwick) [inline index.html]: Good suggestion. Manu can you change this please

**Comment by David-Chadwick** [comment]
Good suggestion. Manu can you change this please?

David

**Comment by msporny** [comment]
> Good suggestion. Manu can you change this please?

Done.

**Comment by msporny** [comment]
ping @dlongley @talltree @nage - review and comment, please.

**Comment by kimdhamilton** [comment]
tiny typo: "Deptartment of Computing"

**Pull request merged**